packages/SwingSet/src/kernel/vat-loader/manager-helper.js

```diff
@@ -264,6 +265,13 @@ function makeManagerKit(
     return vres;
   }
 
+  /**
+   * @returns { boolean }
+   */
+  function getEnableFakeDurable() {
```

Comment:
I'm not sure routing this query through `manager-helper.js` is necessary (or adds much). All the manager types already have access to `kernelKeeper`.. could we just have them call `kernelKeeper.getEnableFakeDurable()` directly?

Reply:
That was what I tried to do initially, but there's one case where `kernelKeeper` is not in scope at the point where it's needed.

Reply:
counterproposal: https://github.com/Agoric/agoric-sdk/pull/5529